hera/R/zzz.R

```diff
@@ -37,25 +37,182 @@ print_vignette <- function(x, ...) {
 
 NAMESPACE <- environment()
 the <- NULL
+original_file <- base::file
+
+xeus_download_file <- function(
```

Comment:
Would it be possible to move the logic for downloading to its own *download.R* file?